snapcraft/internal/sources/errors.py

```diff
@@ -1,6 +1,6 @@
 # -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
 #
-# Copyright (C) 2015-2017 Canonical Ltd
+# Copyright (C) 2015-2017, 2019 Canonical Ltd
```

Comment:
I'm unsure if copyright notices must have this year-by-year granularity, or just start to most recent modification is enough. I use the latter in my works.

Reply:
Oh, we had a long discussion with legal on how to do it, maybe I can pull out the thread